client/landing/jetpack-cloud/components/threat-dialog/index.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Dialog } from '@automattic/components';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	threatId: number;
+	threatTitle: string;
+	threatDescription: string;
+	action: 'fix' | 'ignore';
+	showDialog: boolean;
+	onCloseDialog: Function;
+}
+
+class ThreatDialog extends React.PureComponent< Props > {
+	performAction = () => {
```

Comment:
I think we should pass this this as well. The same way we are passing in the onCloseDialog function.